package.json

```diff
@@ -1,51 +1,28 @@
 {
-  "name": "root",
+  "name": "asn1-schema-monorepo",
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "devDependencies": {
     "@types/mocha": "^8.0.0",
     "@types/node": "^13.11.1",
     "lerna": "^3.22.1",
     "mocha": "^8.0.1",
-    "rimraf": "^3.0.2",
```

Comment:
`rimraf` is required for `rebuild` scripts. Does it work?

Reply:
Yes, It does.

![image](https://user-images.githubusercontent.com/9417251/89996509-8f421f80-dc93-11ea-9313-9250fb43234d.png)